test/unit/address-book-controller.js

```diff
@@ -19,35 +19,39 @@ const mockKeyringController = {
 describe('address-book-controller', function () {
   var addressBookController
 
-  beforeEach(function () {
-    addressBookController = new AddressBookController({}, mockKeyringController)
+  beforeEach(async function () {
+    addressBookController = await new AddressBookController({}, mockKeyringController)
```

Comment:
@tmashuang seems to have just put `await` in front of everything -- theres like 200 other examples in this PR